catalog/ui/catalog-ui-search/src/main/webapp/component/select/select.view.js

```diff
@@ -41,11 +41,21 @@ define([
                 'data-help': this.model.get('help')
             };
         },
+        onRender: function(){
+            if (this.model.get('description')) {
+                this.$el.attr('data-help', this.model.get('description'));
+            }
+        },
         serializeData: function(){
             var modelJSON = this.model.toJSON();
             if (modelJSON.label.constructor === Array){
                 modelJSON.label = modelJSON.label.join(' | ');
             }
+            if (modelJSON.description) {
+                // add line breaks to separate the description from the label
+                // within the tooltip
+                modelJSON.description = '\n\n' + modelJSON.description;
```

Comment:
:question: If something is made to use this description in the future, they will also receive the two extra line breaks. Is this tightly coupled to ensure that this is only used for the tooltip in which this formatting trick is needed?